### DISCUSSION TITLE
NextcloudPi : how to change IP adress defined automatically during wizard?

### DISCUSSION THREAD

**Discussion by jokerigno** [tteck/Proxmox#1146] [Help Needed] [answered]
Hi,

not a issue, I think. I just installed nextcloud using your tool (thanks) and NextcloudPi is reacheble with an ip adress 192.168.2.157.

How can I change it to something different like 192.168.2.203?

Thanks

**Comment by tteck** [comment, marked as answer]
In the Proxmox GUI
1) click on "nextcloudpi"
2) Click on the "Network" tab.
3) Select the "Edit" option.
4) Set the IP address you want to use as static.

![Screenshot 2023-03-10 10 02 26 AM](https://user-images.githubusercontent.com/21342670/224350954-d21a1447-5039-4590-89be-ad112922ca4a.png)

Instead of manually setting the IP address as static after creating the container, you can use advanced settings during container creation to configure the network settings and set the IP address as static.

**Reply by jokerigno** [reply]
OK I've done linke you suggested but I think there's still one piece missing.
After reboot I see an error message:


**Access through untrusted domain**

Please contact your administrator. If you are an administrator, edit the "trusted_domains" setting in config/config.php like the example in config.sample.php.

Further information how to configure this can be found in the [documentation](https://docs.nextcloud.com/server/25/go.php?to=admin-trusted-domains).

______________


How can I access via ssh? I tried using putty with ncp as user and password but I receive an error message and suddendly page is closed
